front/app/api/mentions/getMentions.ts

```diff
@@ -0,0 +1,21 @@
+import fetcher from 'utils/cl-react-query/fetcher';
+import mentionsKeys from './keys';
+import { IMentions, IQueryParameters, MentionsKeys } from './types';
+import { queryClient } from 'utils/cl-react-query/queryClient';
+import { CLErrors } from 'typings';
+
+export const fetchMentions = (queryParams: IQueryParameters) =>
+  fetcher<IMentions>({
+    path: `/mentions/users`,
+    action: 'get',
+    queryParams,
+  });
+
+const getMentions = (queryParameters: IQueryParameters) => {
```

Comment:
We have a little bit of a different approach here creating a method, instead of a hook. The reason is that the `Mention` component has a prop `data` which simultaneously provides the query and a callback to display the suggestions. To make it work, we need this method to fetch the suggestions and pass them to the callback method all in one go :)